app/views/foreman_virt_who_configure/configs/steps/_general_information_form.erb

```diff
@@ -1,4 +1,12 @@
-<div class="tab-pane <%= show_partial_wizard(step) %>" id="config_general_information">
-  <!-- TODO: remove placeholder text -->
-  General information
+<div class="tab-pane <%= show_config_partial_wizard(step) %>" id="config_general_information">
+  <% if @config.organization_id.present? %>
+    <%= select_f f, :organization_id, Organization.authorized(:view_organizations), :id, :to_s, {}, :label => _('Owner'), :disabled => true %>
+    <%= f.hidden_field :organization_id %>
+  <% else %>
+  <%= select_f f, :organization_id, Organization.authorized(:view_organizations), :id, :to_s, {}, :label => _('Owner') %>
+  <% end %>
+  <%= select_f f, :hypervisor_type, ForemanVirtWhoConfigure::Config::HYPERVISOR_TYPES, :first, :last %>
+  <%= text_f f, :hypervisor_server %>
+  <%= text_f f, :hypervisor_username %>
+  <%= password_f f, :hypervisor_password %>
```

Comment:
I found out password isn't provided as default value in the form and therefore moving from 2nd to 3rd step in the wizard re-sets it to blank string.